GPS.Service/Conrollers/QueryOutagesController.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using GPS.Data;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+
+namespace GPS.Service.Conrollers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class QueryOutagesController : Controller
```

Comment:
Might not have specified, but we want the backend data service to be a separate service from the UI web app.  There will be other consumers of the data besides the UI frontend